src/main/java/HW1/HW1_3.java

```diff
@@ -0,0 +1,56 @@
+package HW1;
+
+/*
+3.	Write a Java program to print the sum (addition), multiply, subtract, divide and remainder of two numbers. (Create a separate method for each operation)
+Test Data:
+Input first number: 125
+Input second number: 24
+Expected Output :
+125 + 24 = 149
+125 - 24 = 101
+125 x 24 = 3000
+125 / 24 = 5
+125 mod 24 = 5
+ */
+public class HW1_3 {
+    public String getText1(String a, String b, String c) {
+        return a + " + " + b + " = " + c;
+    }
+    public String getText2(String a, String b, String c) {
+        return a + " - " + b + " = " + c;
+    }
+    public String getText3(String a, String b, String c) {
+        return a + " * " + b + " = " + c;
+    }
+    public String getText4(String a, String b, String c) {
+        return a + " / " + b + " = " + c;
+    }
+    public String getText5(String a, String b, String c) {
+        return a + " % " + b + " = " + c;
+    }
+
+    static String sum() {
```

Comment:
expected values like this is better to store in the test class. let's change existing methods from above (getText1, getText2...) to be sum(), subtract()... and return the value of this methods (String c, should not be the method's argument, but the result of some operations). 

You can also just return the value of this operation and compare it with expected, so your test class will look like this - this should be even better solution:
`Assertions.assertEquals(101, Task3.subtract(125, 24), "Incorrect result of subtraction");`